internal/app/exporter/prometheus.go

```diff
@@ -0,0 +1,14 @@
+package exporter
+
+import (
+	"regexp"
+)
+
+func ClearPrometheusLabel(str string) string {
```

Comment:
```suggestion
func clearPrometheusLabel(str string) string {
```
Only used in this package, I would not export it.

Reply:
Done, see https://github.com/qonto/prometheus-rds-exporter/pull/151#discussion_r1537362418